test/common/index.js

```diff
@@ -45,3 +45,21 @@ got ${it.actual}
 ${it.stack}`);
   });
 }
+
+function expectsError(fn, exact) {
+  function innerFn(error) {
```

Comment:
I'm wondering if `assert.throws` works exactly the same with this function?

Reply:
Yes, because I wanted to do a more perfection assert in this function just like in Node, but there are some operations that are not supported in ShadowNode, so I reserve it. ：)
Next I will revamp that api, I will do it!

Reply:
Thanks!